### ISSUE TITLE
Trouble accessing WP container through reverse-proxy

### ISSUE THREAD

**Issue by sadrian80** [bitnami/bitnami-docker-wordpress#350]
I'm trying to deploy multiple bitnami/wordpress containers. There a separate nginx reverse proxy running. Access to the bn-containers via IP:port works as expected but I can't access them through the reverse proxy.

### Name and Version
bitnami/wordpress:6

### What steps will reproduce the bug?
- deploy bitnami/wordpress:6
- configure reverse-proxy in nginx (separate) to route xyz.com to https://wordpress-ip:443

### What is the expected behavior?
Wordpress is accessible through the IP:Port of the deployed container.
I would expect to access Wordpress through xyz.com.

### What do you see instead?
I get a 504: Gateway Timeout

### Additional information
The container is untouched.
For testing I'm using a simple nginx conf:
```
server {
  set $forward_scheme http;
  set $server         "10.30.0.172";   [<- my docker host ip]
  set $port           80;
  listen 80;
  server_name xyz.com;
}


```

**Comment by javsalgar** [comment]
Hi,

It seems to me that the issue may rely on the proxy configuration or the container networking, not the Bitnami configuration as it is compatible with proxies like, for example, Ingress. I see that you are setting a static IP, which may have changed due to the ephemeral nature of containers. I will leave the ticket open in case someone from the community wants to participate with their experience.

**Comment by sadrian80** [comment]
Update:
I did further troubleshooting with help from the nginx community, but the reverse proxy seemed to be working fine. In the end, someone in the portainer discord had comparable issues. The solution was to wipe and reinstall the VM that is the docker host – now the networking runs as intended.

We initially set up the host with Portainer 2.11. There might have been issues that got resolved since then.

Thanks again @javsalgar for letting this issue open. Others might have closed it down before even answering! But now it is …

Solved.
